Fix broken link (#576)
numtide/treefmt: README.md

- gofmt for Golang
- Prettier for JavaScript/HTML/CSS

> Find the full list of supported formatters [here](https://treefmt.com/latest/getting-started/configure/#supported-formatters).

Reviewer: The linked docs don't actually list supported formatters:

> Any formatter that follows the [spec](https://treefmt.com/latest/reference/formatter-spec/) is supported out of the box.
> 
> Already 60+ formatters are supported.
> 
> To find examples, take a look at https://github.com/numtide/treefmt-nix/tree/main/examples.
> 
> If you are a Nix user, you might also like https://github.com/numtide/treefmt-nix, which uses Nix to pull in the right formatter package and seamlessly integrates both together.

If there isn't an official/canonical list shall we just tweak the link's wording?

```suggestion
You can find information on supported formatters [here](https://treefmt.com/latest/getting-started/configure/#supported-formatters).
```

Author: Good call, done!
